build-static.sh

```diff
@@ -45,29 +45,34 @@ elif ! docker ps >/dev/null ;then
 fi
 
 PKG=$1
-
+TARGET_ARCH=$2
```

Comment:
```suggestion
TARGET_ARCH=${2-$ARCH}
```
To avoid having an error if `$2` isn't defined